prowler/lib/mutelist/mutelist.py

```diff
@@ -109,21 +113,49 @@ def parse_mutelist_file(session: Session, aws_account: str, mutelist_path: str):
 
 
 def mutelist_findings(
-    mutelist: dict,
-    audited_account: str,
+    global_provider: Any,
     check_findings: list[Any],
 ):
     # Check if finding is muted
     for finding in check_findings:
```

Comment:
Add a TODO here to move this mapping to the execute_check function and pass that output to the mutelist and the report, thanks!